index.html

```diff
@@ -4975,6 +4975,637 @@ <h2 id="AudioWorkerParamDescriptor">
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
+            </dt>
+            <dd>
+              <p>
+                This array contains <a>AudioParamDescriptor</a>s, which are
+                descriptors for each controllable parameter on an
+                <a>AudioNode</a>. This static property is referenced in the
+                construction of an <a>AudioWorkletNode</a> to create instances
+                of an <a>AudioParam</a> in the node.
+              </p>
+            </dd>
+            <dt>
+              readonly attribute AudioContextInfo contextInfo
+            </dt>
+            <dd>
+              <p>
+                This getter returns an <a><code>AudioContextInfo</code></a>
+                object from the associated <a>AudioContext</a>.
+              </p>
+            </dd>
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The <code>onmessage</code> handler is called whenever the
+                <a><code>AudioWorkletNode</code></a> posts a message back to
+                the corresponding processor object.
+              </p>
+            </dd>
+            <dt>
+              void process()
+            </dt>
+            <dd>
+              <p>
+                This method gets executed isochronously by the audio
+                <a>rendering thread</a>.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  Float32Array[][] inputs
+                </dt>
+                <dd>
+                  An Array of Arrays of <code>Float32Array</code>s. The top-
+                  level Array is organized by input; each input may contain
+                  multiple channels; each channel contains a
+                  <code>Float32Array</code> of sample data. The initial size of
+                  the channel array will be determined by the number of
+                  channels by the <a><code>channelCount</code></a> property in
+                  the <a><code>AudioWorkletNodeOptions</code></a> object
+                  supplied to the <a><code>AudioWorkletNode</code></a>
+                  constructor. However, the number of channels may be altered
+                  dynamically by the number of channels in the incoming audio
+                  stream.
+                </dd>
+                <dt>
+                  Float32Array[][] outputs
+                </dt>
+                <dd>
+                  An Array of Arrays of <code>Float32Array</code>s. The top-
+                  level Array is organized by output; each output may contain
+                  multiple channels; each channel contains a
+                  <code>Float32Array</code> of sample data.
+                </dd>
+                <dt>
+                  Object parameters
+                </dt>
+                <dd>
+                  This object attribute exposes a correspondingly-named
```

Comment:
I find it really hard to figure out what parameter really is.  I discussed this with @hongchan to understand what this is.  I think the following text is a bit easier to understand:

`parameters` provides an association between the names of an AudioParams specified by the AudioParamDescriptors during construction and the automation values for the AudioParam. Thus, `parameters[name]` returns a Float32Array of the automation values corresponding to the AudioParam named "name".

Probably still need a bit more text, but I think this captures what parameters really contains.